common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -69,28 +75,58 @@ hfe_initializer_egypt_main = {
 			max = 270
 		}
 		init_effect = {
+			save_global_event_target_as = hfe_egypt_home_of_light
 			create_country = {
 				name = "Ju Ahet"
 				type = human_fallen_empire
 				name_list = cybrxkhan_fantasy_egyptian
-				ship_prefix = ""
+				ship_prefix = "Ra"
 				ignore_initial_colony_error = yes
 				species = event_target:hfe_egypt_species
+				authority = auth_imperial
 				ethos = {
 					ethic = ethic_fanatic_spiritualist
+					ethic = ethic_authoritarian
 				}
-				authority = auth_imperial
 				civics = {
 					civic = hfe_civic_egypt_genetic
 					civic = hfe_civic_egypt_rulers
 				}
+				origin = hfe_civic_egypt_rulers_origin
 				effect = {
+					save_global_event_target_as = hfe_egypt_country
+					set_graphical_culture = reptilian_01 # Learned from Ra'ar
+					set_city_graphical_culture = fallen_empire_03 # Light blue/red/yellow
 					set_country_flag = hfe_egypt
 					set_country_flag = hfe_assimilator_discovered
 					set_country_flag = hfe_marked_as_human
-					give_technology = {
-						tech = tech_titans
-					}
+					hfe_effect_init_give_tech_t5 = yes
+					give_technology = { message = no tech = tech_titans }
+					give_technology = { message = no tech = tech_colossus }
+					give_technology = { message = no tech = tech_pk_cracker }
+#					give_technology = { message = no tech = tech_pk_neutron }
+					give_technology = { message = no tech = tech_psi_jump_drive_1 }
+					give_technology = { message = no tech = tech_psionic_shield }
+					give_technology = { message = no tech = tech_gene_tailoring }
+					give_technology = { message = no tech = tech_genome_mapping }
+					give_technology = { message = no tech = tech_vitality_boosters }
```

Comment:
Shouldnt they have all this tech?

Reply:
No. These include the ones that come with the Ascension Perk. Without forcing the game to give it to them, the Ju Ahets wouldn't have every genetic ability given to them.

Reply:
Still a lot of tech they already have from country type

Reply:
The Ju Ahet cannot get the Ascension Perk Technology without it being forced by Event, or them actually taking the Perk.